src/InitialConditions/baro_wave.jl

```diff
@@ -129,10 +129,30 @@ function center_initial_condition_baroclinic_wave(
     end
     # TODO: Include ability to handle nonzero initial cloud condensate
 
+    precip_kwargs =
+        if precip_model isa NoPrecipitation &&
```

Comment:
If we use 0-moment microphysics we also should not add q_rai and q_sno to variable list